docs/integrations/sources/yahoo-finance-price.md

```diff
@@ -0,0 +1,9 @@
+# Yahoo Finance Price
+
+The Airbyte Source for [Yahoo Finance Price](https://finance.yahoo.com/)
+
+## Changelog
+
+| Version | Date       | Pull Request                                             | Subject                       |
+| :------ | :--------- | :------------------------------------------------------- | :---------------------------- |
+| 0.1.3   | 2022-03-23 | [10563](https://github.com/airbytehq/airbyte/pull/10563) | 🎉 Source Yahoo Finance Price |
```

Comment:
This connector is not published to docker hub. I think we should mention it in the doc.